src/mongocxx/collection.cpp

```diff
@@ -426,6 +430,8 @@ stdx::optional<bsoncxx::document::value> collection::find_one_and_replace(
         flags |= ::MONGOC_FIND_AND_MODIFY_RETURN_NEW;
     }
 
+    // TODO: use options.max_time() when available in the C driver
```

Comment:
Please file a new CXX driver ticket in the C++11 driver epic, and set the fix version to "waiting on dependency", and add the "waiting-on-mongoc" label. The ticket should be linked as "depends on" the relevant C driver ticket. Also, how should we note this in the header comments? Or should this be a relnote?